Dockerfile

```diff
@@ -15,11 +15,9 @@ WORKDIR /${TMX_PATH}
 
 COPY package*.json ./
 
-RUN npm ci --only=prodution
+RUN npm ci --only=production --ignore-scripts
```

Comment:
Has `ignore-scripts` some impact on the analysis process of the build?

Reply:
No, this script prevents malicious command execution in CI. (prestart)

Reply:
Ok! :+1: